manifests/a/AtomicJar/Testcontainers/1.4.16/AtomicJar.Testcontainers.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.2.10 $debug=NVS1.CRLF.5-1-22621-1778.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: AtomicJar.Testcontainers
+PackageVersion: 1.4.16
+InstallerLocale: en-US
+InstallerType: wix
+Scope: user
+InstallModes:
+- silent
+- interactive
+- silentWithProgress
+InstallerSwitches:
+  Silent: /quiet
+  SilentWithProgress: /passive
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://app.testcontainers.cloud/download/testcontainers-desktop_windows_x86-64.msi
+  InstallerSha256: 8B21F4065D0E7A80C59085032C17DFD84BAC3738A3E8B50ECF3332F6B42E38BA
+  ProductCode: '{458CD054-E434-48E9-917D-B5787CF8AB7E}'
+- Architecture: x86
+  InstallerUrl: https://app.testcontainers.cloud/download/testcontainers-desktop_windows_x86-64.msi
+  InstallerSha256: 8B21F4065D0E7A80C59085032C17DFD84BAC3738A3E8B50ECF3332F6B42E38BA
+  ProductCode: '{458CD054-E434-48E9-917D-B5787CF8AB7E}'
```

```suggestion
- Architecture: x86
  InstallerUrl: https://app.testcontainers.cloud/download/testcontainers-desktop_windows_x86-64.msi
  InstallerSha256: 8B21F4065D0E7A80C59085032C17DFD84BAC3738A3E8B50ECF3332F6B42E38BA
  ProductCode: '{458CD054-E434-48E9-917D-B5787CF8AB7E}'
```
Hi @licon4812,

`x86` validations are performed on a 32-bit OS, so I believe that this might be a 64-bit program.